addons/godot-xr-tools/staging/staging.gd

```diff
@@ -269,12 +284,12 @@ func load_scene(p_scene_path : String, user_data = null) -> void:
 ## unless hidden.
 func set_fade(p_value : float):
 	if p_value == 0.0:
-		$Fade.visible = false
+		fade.visible = false
```

Comment:
I think you lost a change here. The variable fade is not defined anywhere or set anywhere.